### PR TITLE
Add Skyline QAPortal help page

### PR DESCRIPTION
**Pull request by SkylineCookieMonster**

### CODE REVIEW

File: connector/doc/Skyline_QAPortal.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+There is an option to push the results to the **Skyline QA Portal**. This will give **Skyline Communications** visibility on your tests. To enable this:
+
+1. Make sure you have a mail server configured on your DMS, so the results can be mailed to the portal
+
+1. Configure the **API Key** and the **Client ID** on the **Settings** page of the Skyline QAPortal element.
+
+1. Enable the option **Forward Results to Portal**.
+
+The API Key and Client ID will need to be provided by Skyline and need to be kept **secret** to your organization.
```
Comment (MariekeGO) [inline connector/doc/Skyline_QAPortal.md:39]: I think we should make this a bit more clear. How do they need to acquire the API Key and client ID? And what do you mean with "kept secret to your organization"?

File: connector/doc/Skyline_QAPortal.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+To add a regression test, go to the **General** page, right-click the **Regression Tests** table, and select **Add new test**.
+
+If you have just added a new regression test, you may need to use the **Refresh Tests** button to see it in the dropdown box.
+
+### Adding Input Parameters
+
+If your regression test has input parameters, configure these in the **Input** column of the **Regression Tests** table.
+
+If you have not yet define any **input group** (i.e. the dropdown is empty), create a new input group via **...** > **New**. This will create a new group **Input Group** in the **Input Parameters** table. You will need to configure the **value** in that table. You can also use the right-click menu of that table to create or remove input groups.
```
Comment (MariekeGO) [inline connector/doc/Skyline_QAPortal.md:60]: Where can users find that "..."? I think we should make this a bit more clear, so they don't have to go searching.

Comment (SkylineCookieMonster) [reply connector/doc/Skyline_QAPortal.md]: https://github.com/SkylineCommunications/dataminer-docs-connectors/assets/107162106/22b8b4ab-34b2-4975-93cb-e961ed880635

The video shows how to Add/Change the Input Parameter Groups. Maybe we can add the video to make it more clear?

Comment (SkylineCookieMonster) [reply connector/doc/Skyline_QAPortal.md]: @MariekeGO Let me know if you want me to still change anything to the text here.

Comment (MariekeGO) [reply connector/doc/Skyline_QAPortal.md]: I tried embedding the video, but for some reason this did not work. I've now instead added a more detailed description with a screenshot. Does this look OK to you?

Comment (SkylineCookieMonster) [reply connector/doc/Skyline_QAPortal.md]: @MariekeGO, looks good to me. Thanks for updating!

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**